specifyweb/specify/migrations/0027_CO_children.py

```
@@ -2,95 +2,25 @@
-
-                for item in items:
-                    item.ishidden = False
-                    item.save()
-
+from specifyweb.specify.migration_utils import update_schema_config as usc
 
 class Migration(migrations.Migration):
 
     dependencies = [
         ('specify', '0026_taxontreedef_alter_delete'),
     ]
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)